zeth-contracts/contracts/Pghr13Mixer.sol

```diff
@@ -3,6 +3,7 @@
 // SPDX-License-Identifier: LGPL-3.0+
 
 pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
```

Comment:
The use of this encoder is unsafe for this version of solidity (several severe bugs have originated from this encoder in the past). It is only starting 0.6.0 that this encoder can be considered safe (https://solidity.readthedocs.io/en/v0.6.0/layout-of-source-files.html#abiencoderv2).

Do you really need to use it here?

Reply:
Yes, exactly.  It's more consistent with a single event (so the signature of the event fn is cleaner).
Although we can revert if until 0.6.0 if you prefer.

Reply:
I see, thanks.
We can keep the encoder here for now, but this is another reason to switch to v0.6.0 of solidity

Reply:
Related to https://github.com/clearmatics/zeth/issues/149